### PR TITLE
sync api docs with website

### PR DESCRIPTION
**Pull request by kellyselden**
Blocked on https://github.com/ember-cli/ember-cli.github.io/pull/130

### CODE REVIEW

File: lib/models/blueprint.js [hunk 1]
```diff
@@ -59,14 +59,14 @@ const writeFile = RSVP.denodeify(fs.outputFile);
   `files` contains templates for the all the files to be
   installed into the target directory.
 
-  The `__name__` token is subtituted with the dasherized
+  The __`__name__`__ token is subtituted with the dasherized
```
Comment (Turbo87) [inline lib/models/blueprint.js]: this looks strange...

Comment (Turbo87) [inline lib/models/blueprint.js]: also this should probably by `substituted` instead of `subtituted`, right?

Comment (trabus) [inline lib/models/blueprint.js]: I think that's just for bolding it __`__name__`__ vs `__name__`

Comment (Turbo87) [inline lib/models/blueprint.js]: well, I understand what it does, but what is not clear to me is why we would want it explicitly bold here. we could do the same using CSS if we wanted to.

**Comment by Turbo87** [comment]
@kellyselden can you update and rebased this PR?

**Comment by Turbo87** [comment]
@kellyselden friendly reminder 😉

**Comment by Turbo87** [comment]
closing due to inactivity and conflicts

**Pull request closed without merging**